package.json

```diff
@@ -52,6 +53,7 @@
     "request": "^2.88.0",
     "semver": "^5.6.0",
     "shx": "^0.3.2",
+    "sinon": "^9.0.1",
```

Comment:
These changes should be in spec-main/package.json

Reply:
I've moved the sinon dependencies into spec-main/ in 839eebe12f4721c6f0d3e2f5f2bfefe1b8dce9eb

I'm a little unclear on the division of dependencies? The change you suggested is what makes intuitive sense to me, but I don't undersatnd why the top-level package.json is where the dependencies on @types/chai, @types/chai-as-promised, @types/dirty-chai, @types/mocha, eslint-plugin-mocha are specified...?

Reply:
The way I see it is that the dependencies should be located where they are used.  If we are using them in multiple places then it makes sense to put them at the top level.